packages/hardhat/contracts/FundingVault.sol

```diff
@@ -104,9 +126,11 @@ contract FundingVault{
         uint256 _blockLimit,
         uint256 _exchangeRate,
         address _withdrawlAddress,
-        address _developerFeeAddress, //develper address
+        address _developerFeeAddress, 
```

Comment:
Remove unnecessary comments before the constructor. Move those comments to the right side of the variable declarations instead in lines 55--67.

Reply:
Fixed